tests/specs/lint/lint_plugin_comments/comments.out

```diff
@@ -0,0 +1,67 @@
+{
+  program: [
+    { type: "Line", range: [ 0, 14 ], value: " before line" },
+    {
+      type: "Block",
+      range: [ 15, 38 ],
+      value: "*\n * before block\n "
+    },
+    { type: "Line", range: [ 58, 72 ], value: " inside line" },
+    {
+      type: "Block",
+      range: [ 75, 102 ],
+      value: "*\n   * inside block\n   "
```

Comment:
Is it intentional for the value property to omit the leading `/*` and trailing `*/` of block comments?

Reply:
Yes, see https://typescript-eslint.io/play/#ts=5.7.2&showAST=es&fileType=.ts&code=PQKhCgAIUgBAHAhgJ0QW0gbwM4BdkCWAdgOYC%2BkAZgPbVQjDjDCQBGK4lArkQMa4FqRKrQAUNagEos4cGSA&eslintrc=N4KABGBEBOCuA2BTAzpAXGYBfEWg&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA&tokens=false